app/core/api/background-job.js

```diff
@@ -1,14 +1,22 @@
 const fetchJson = require('./fetch-json')
 
-function create (jobType, input) {
-  return fetchJson('/db/background-jobs', {
+function create (jobType, input, other = undefined) {
+  let url = '/db/background-jobs'
+  if (other) {
+    url = '/' + other + url
+  }
+  return fetchJson(url, {
     method: 'POST',
     json: { type: jobType, input }
   })
 }
 
-function get (jobId, options = {}) {
-  return fetchJson(`/db/background-jobs/${jobId}`, options)
+function get (jobId, options = {}, other = undefined) {
```

Comment:
not a fan of 2 optional params in same function. Don't need to mark as undefined since that would be the case anyway.
`function get (jobId, other, options = {})`